dkc/core/views.py

```diff
@@ -12,14 +12,22 @@ def staff_home(request: HttpRequest) -> HttpResponse:
 
 
 @staff_member_required
-def staff_tree_size(request: HttpRequest) -> HttpResponse:
+def staff_tree_list(request: HttpRequest, sort_by) -> HttpResponse:
+    try:
```

Comment:
@danlamanna Should I use query params here? This way is easier to build URLs with, it's just slightly annoying that I can't easily specify a default (without creating a separate URL redirect).

Reply:
This definitely feels more like it should be a query param. (I could see us adding more parameters in the future.)

Reply:
Fixed.